practice/src/js/helpers/api.js

```diff
@@ -22,6 +22,31 @@ const APIHelper = {
       throw new Error(`Get data fail ${error.message}`);
     }
   },
+
+  /**
+   * Sends a DELETE request to the API to delete data.
+   * @param {string} endpoint - The API endpoint for deleting data.
+   * @returns {Promise<Object>} - The JSON response from the server indicating success or failure.
+   */
+  async delete(endpoint) {
+    try {
+      const response = await fetch(`${API.BASE_URL}/${endpoint}`, {
+        method: 'DELETE',
+      });
+
+      if (!response.ok) {
+        throw new Error('Failed to delete data');
+      }
+
+      return {
+        isSuccess: true,
+      };
+    } catch (error) {
+      return {
+        isSuccess: false,
+      };
+    }
+  },
 };
 
 export { APIHelper };
```

Comment:
I suggest using `HttpClient` instead of `APIHelper`

Reply:
One more question, why we don't use class in this case?

Reply:
because using an object provides a simple and straightforward way to encapsulate related functionality (get, put, post, delete methods) without the added complexity of a class

Reply:
@JennyDoJD No, it is not wrong when using an object but it's not a good way, an HTTP client service does many things inside, some private methods for special cases, or sometimes you want to inherit this service for other services....you can refer some HTTPClient file to learn more